### PR TITLE
CLM-27746: Renamed Grandfathering Prop

### PR DESCRIPTION
**Pull request by entroform**
https://sonatype.atlassian.net/browse/CLM-27746

### Testing done

```[tasklist]
### Submitter checklist
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

**Comment by vbazurtob** [comment]
LGTM

+1

**Review by EmilyBermudez88 — APPROVED** [review]
Once we start testing I think we'll have to look at running this with an older version of IQ to make sure `legacyViolationCount` can still pull `grandfatheredPolicyViolationCount`. Otherwise LGTM!

File: src/main/java/org/sonatype/nexus/ci/iq/PolicyEvaluationHealthAction.groovy [hunk 1]
```diff
@@ -78,7 +78,7 @@ class PolicyEvaluationHealthAction
     this.criticalPolicyViolationCount = policyEvaluationResult.criticalPolicyViolationCount
     this.severePolicyViolationCount = policyEvaluationResult.severePolicyViolationCount
     this.moderatePolicyViolationCount = policyEvaluationResult.moderatePolicyViolationCount
-    this.legacyPolicyViolationCount = policyEvaluationResult.grandfatheredPolicyViolationCount
+    this.legacyPolicyViolationCount = policyEvaluationResult.legacyViolationCount
```
Comment (guillermo-varela) [inline src/main/java/org/sonatype/nexus/ci/iq/PolicyEvaluationHealthAction.groovy:81]: The `legacyViolationCount` will be available on the next version of `nexus-platform-api`.
So when it's published we'll need to update the dependency at:
https://github.com/jenkinsci/nexus-platform-plugin/blob/22c94fa171e925046874b92fb5cff73239e62ca9/pom.xml#L86

**Review by guillermo-varela — APPROVED** [review]
+1

**Pull request merged**